### DISCUSSION TITLE
Less repetition, please, and truncate long items

### DISCUSSION THREAD

**Discussion by RhvdG** [vslinko/obsidian-zoom#90] [Ideas]
<img width="631" alt="image" src="https://user-images.githubusercontent.com/34862159/230626419-0a3e23c0-34c0-47bf-874f-6755155561ec.png">

The title of my note now appears three times: once on the tab bar, a second time as the file name in the zoom sequence, and a third time as the (user generated) H1 header. I would prefer to lose at least the third mention, but preferable the second one as well (optionally). 

More importantly, the item that I invoked the zoom-in command on ("Here's the third item") appears twice. I would prefer to lose it in the zoom sequence, and keep it in the main part of the window (again, optionally). 

There are probably other ways to reduce these repetitions. They become especially unwieldy with longer items (i.e. more text). 

Truncating long items using ellipses would be a good thing, too. Now the longer item just runs off the leaf. Having such long strings in the zoon sequence defeats the purpose, as the zoom sequence takes up almost as much space as the same content in original list layout.

<img width="667" alt="image" src="https://user-images.githubusercontent.com/34862159/230627695-d16b1c3b-f068-4ee6-82b3-521c9c218d58.png">

**Comment by FelipeRearden** [comment]
Hello @RhvdG 

I am gonna try to help sharing my CSS snippet with you, Maybe this will help you too 🙏

```css
/* Obsidian Zoom */

.zoom-plugin-title:first-child {
	display: none;
}

.zoom-plugin-delimiter:first-of-type {
	display: none;
}
```

I hope it helps 🙏

**Reply by RhvdG** [reply]
Thanks a lot, I appreciate it! I went ahead and tried to shave off the remaining repetition (the item zoomed in on is seen both in the body text and in the zoon sequence) but I couldn't get that done.

**Reply by FelipeRearden** [reply]
I am pretty bad at CSS but my suggestion to you is to take a look at these docs here: 

→ https://www.w3schools.com/cssref/css_selectors.php

Maybe you could use one of these: 

![075F6B39-01FA-4F35-B7D2-1AE6938240AA](https://user-images.githubusercontent.com/85180696/230726572-956a5712-83d8-45ce-8b6d-6ac1da3483b6.jpeg)

Maybe you could target using one of these selectors 

I wish you good luck :)

**Reply by RhvdG** [reply]
Thanks again! Giving it a try...

**Comment by rjst** [comment]
Just an opinion - I love it just the way it is. Breadcrumbs and in-document info have different uses. If you change, can you make it optional ? 🙏🏻😀
